zip-0230.rst

```diff
@@ -0,0 +1,352 @@
+|                                    |                          |                                        |at some block height in the past.                                          |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``192 * nSpendsSapling``            |``vSpendProofsSapling``   |``byte[192 * nSpendsSapling]``          |Encodings of the zk-SNARK proofs for each Sapling Spend.                   |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``64 * nSpendsSapling``             |``vSpendAuthSigsSapling`` |``byte[64 * nSpendsSapling]``           |Authorizing signatures for each Sapling Spend.                             |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``192 * nOutputsSapling``           |``vOutputProofsSapling``  |``byte[192 * nOutputsSapling]``         |Encodings of the zk-SNARK proofs for each Sapling Output.                  |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``64``                              |``bindingSigSapling``     |``byte[64]``                            |A Sapling binding signature on the SIGHASH transaction hash.               |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+| **Orchard Transaction Fields**                                                                                                                                                     |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``varies``                          |``nActionsOrchard``       |``compactSize``                         |The number of Orchard Action descriptions in                               |
+|                                    |                          |                                        |``vActionsOrchard``.                                                       |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``852 * nActionsOrchard``           |``vActionsOrchard``       |``ZSAOrchardAction[nActionsOrchard]``   |A sequence of ZSA Orchard Action descriptions, encoded per                 |
```

Comment:
**OrchardZsaAction** ... A sequence of **Orchard-ZSA** Action...

Reply:
Done